Gemfile

```diff
@@ -8,6 +8,7 @@ gem "blacklight_range_limit", "~> 6.0"
 gem "bootstrap-sass", "~> 3.4"
 gem "bundler", "~> 2.0"
 gem "bunny"
+gem "dalli"
```

Comment:
How is this used? I don't see the setup as documented in https://github.com/petergoldstein/dalli#usage-with-rails-3x-and-4x

Reply:
I think it was Rails 4 or 5 where Rails just started using it if it's available, no config necessary: https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-memcachestore